test/test_yt.py

```diff
@@ -11,14 +13,33 @@
 NO_QUERY_MESSAGE = "You can't look for nothing. !yt <QUERY>"
 
 
+def mocked_search_execute(search_query: str, search_part: str, search_type: str, max_results: int):
+    """
+    Currently only returns a response of video ID's based on max_results.
+    Otherwise returns none.
+    """
+    if search_type == 'video' and search_part == 'id':
+        items = []
+        for _ in range(max_results):
+            # The following line generates an 11 character random string of ascii and digits
```

Comment:
since we don't actually use the ids for anything, you should probably just return a zero padded increasing int or some shit
i.e. `items = [{'id': {'videoId': str(i).zfill(11)}} for i in range(max_results)`
this just makes the code simpler